staging/src/k8s.io/csi-api/pkg/crd/testdata/csinodeinfo.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
I recommend dropping `testdata` from the path for this file. It is currently being used for E2E tests, but we want this to be the authoritative source for the CRD and it should be used by any future "CRD auto install" feature.

Reply:
iirc, various go tools more readily import/preserve non-go files if they are in testdata files, but maybe that requirement is legacy

Reply:
Ack. I'm ok leaving it as is then.